config.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"github.com/getkin/kin-openapi/openapi3"
+	"log"
+)
+
+const passResponseDescription = "PASS"
+
+// loadTestEndpoints loads a default test endpoint request (a GET / request expecting a 200 status code) into an
+// openapi3.Swagger object (see github.com/getkin/kin-openapi).
+func loadTestEndpoints() *openapi3.Swagger {
```

Comment:
should this be `loadTestEndpoints` or `defaultTestEndpoint`?

Reply:
Right now, it's just loading a default test endpoint, but once the custom endpoint functionality gets merged, we're planning for this function to load those endpoints in as well.